scripts/docsgen/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/coder/envbuilder"
+)
+
+func main() {
+	readmePath := "README.md"
+	readmeFile, err := os.ReadFile(readmePath)
+	if err != nil {
+		panic("Error reading " + readmePath + " file")
```

Comment:
```suggestion
		panic("error reading " + readmePath + " file")
```
Using lower case is more idiomatic, see e.g. coder/coder:

```
~/Work/Code/coder main* ⇣
❯ rg 'panic\([a-z]' | wc -l
      94

~/Work/Code/coder main* ⇣
❯ rg 'panic\([A-Z]' | wc -l
       0
```

Reply:
I'm wondering if it has any specific reason for lowercase to be considered more idiomatic.